zqd/handlers_test.go

```diff
@@ -78,7 +78,7 @@ func TestSearchEmptySpace(t *testing.T) {
 	require.Equal(t, "", res)
 }
 
-func TestSearchInvalidRequest(t *testing.T) {
+func TestSearchError(t *testing.T) {
```

Comment:
Thanks for using the zqe.E to create the errors! You should be to add assertions for 400 (bad request) status codes since you specified `zqe.Invalid` in the zqe.E calls.

Reply:
Will do